app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
```

Comment:
It's better to unpack the list which you get with split() like this
`variable_a, variable_b, variable_c = "abc".split()`

Reply:
and then use this variable in your code